src/oph/ehoks/db/db_operations/db_helpers.clj

```diff
@@ -80,13 +80,20 @@
     (jdbc/update! db-conn table values where-clause)))
 
 (defn shallow-delete!
```

Comment:
Ehdotus: Meidän muuten pitäis uudelleennimetä shallow delete soft deleteksi, koska tää delete on nykyään soft mutta ei enää shallow :)  Ei välttämättä API endpointeissa ettei tarvii niitä muuttaa, mutta ainakin koodissa.

Reply:
Nimesin hoksin osalta soft deleteksi, mutta jätin vielä tällä tasolla shallow deleteksi, koska nämä db_helperin funkkarit ovat varsin geneerisiä ja niitä edelleen käytetään yksittäisten hoksin osien poistoon esim patchatessa.

Voidaan mun mielestä kyllä luopua kokonaan shallow-termistä. En osaa sanoa miksi kyseinen termi on aikoinaan valittu, eikä soft delete...